src/screens/Inventory/PurchaseRequest.tsx

```diff
@@ -28,23 +28,25 @@ const styles = (theme: Theme) =>
 interface PurchaseRequestsProps extends RouteComponentProps {
   classes: { content: string; section: string; };
   location: any;
-  product: ProductRecord;
+  // product: ProductRecord;
 }
 
 
 function PurchaseRequest (props: PurchaseRequestsProps) {
-  const { classes, product } = props;
+  const { classes } = props;
   const purchaseRequest: PurchaseRequestRecord = props.location.state.purchaseRequest;
+  const product = useSelector((state: RootState) => selectProductById(state, selectCurrentSiteInventoryById(state, purchaseRequest.inventoryId)?.productId || ""))|| EMPTY_PRODUCT;
```

Comment:
Would you mind doing some redirecting if someone navigates to this URL by typing it in (maybe just redirecting them to inventoryMain)? I'll be doing the equivalent handling in the customer-related pages.

Reply:
Added redirects and updated the PR description with a testable workflow!